tests/target-browser/target_browser_test.go

```diff
@@ -1655,12 +1640,13 @@ var _ = Describe("Target Browser Tests", func() {
 			)
 			BeforeEach(func() {
 				once.Do(func() {
+					removeBackupPlanDir()
 					backupUID = guid.New().String()
 					clusterBackupUID = guid.New().String()
 					createTarget(true)
 					createBackups(noOfClusterBackupPlansToCreate, noOfClusterBackupsToCreatePerClusterBackupPlan, clusterBackupUID, "cluster_backup")
 					createBackups(noOfBackupPlansToCreate, noOfBackupsToCreatePerBackupPlan, backupUID, "backup")
-					backupPlanUIDs = verifyBackupPlansAndBackupsOnNFS(noOfBackupPlansToCreate+noOfClusterBackupPlansToCreate,
+					_ = verifyBackupPlansAndBackupsOnNFS(noOfBackupPlansToCreate+noOfClusterBackupPlansToCreate,
```

Comment:
If not needed, then remove getting empty value here.

Reply:
This is required to verify backup and backupPlan on target.